src/style.css

```diff
@@ -1,3 +1,166 @@
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --white: #fff;
+  --lightgrey: #f3f3f3;
+  --darkgrey: #060b35;
+  --grey: #616070;
+  --blue: #00acdc;
+}
+
+.card {
```

Comment:
card should have padding for all sides to avoid using extra margin for inner elements

Reply:
add padding: 32px 16px 16px; to a card and remove margin-left, margin-right from children